src/components/icon/icon-state.broadcast.ts

```diff
@@ -34,10 +34,22 @@ export class IconsStateBroadcast {
     this.send({
       actionType: ActionType.SyncState,
       collections: this.getUserSetCollection(this.collections).toMap(),
-      references: this.refsCollection.toMap(),
+      references: this.getRefsSetCollection(this.refsCollection).toMap(),
     });
   }
 
+  private getRefsSetCollection(collections: IconsCollection<IconMeta>) {
```

Comment:
For consistency this can also be `getUserRefsCollection` kinda like `getUserSetCollection` just to be clearer what we're filtering for.